### PR TITLE
fix(resolver): Add `env` support

### PR DESCRIPTION
**Pull request by jgkim**
Check out `env` for `plugins`

### CODE REVIEW

**Comment by tleunen** [comment]
Oh great addition @jgkim!

File: src/index.js [hunk 1]
```diff
@@ -5,14 +5,30 @@ const assign = require('object-assign');
...
+
+    if (c && c.config) {
+        let pluginConfig;
+
+        if (Array.isArray(c.config.plugins)) {
+            pluginConfig = c.config.plugins.find(p => p[0] === 'module-alias');
+        }
+
+        if (c.config.env && c.config.env[env] && Array.isArray(c.config.env[env].plugins)) {
+            pluginConfig = c.config.env[env].plugins.find(p => p[0] === 'module-alias');
```
Comment (tleunen) [inline src/index.js]: If babel finds a "default" configuration and a configuration specific to an environment, will it merge these 2 configurations? I believe so.

Would you mind adding a test checking that the right configuration is retrieved when having a specific environment config?

File: src/index.js [hunk 2]
```diff
@@ -5,14 +5,30 @@ const assign = require('object-assign');
 const findBabelConfig = require('find-babel-config'); // eslint-disable-line
 
 function getMappingFromBabel(start) {
+    // eslint-disable-next-line strict
+    'use strict';
```
Comment (tleunen) [inline src/index.js]: This shouldn't be needed ;-)

Comment (jgkim) [inline src/index.js]: Without this, the test fails on Node.js 4 and 5.

File: src/index.js (hunk 1 above)
Comment (jgkim) [inline src/index.js]: I guess babel takes the last one not merging two configs. I'm afraid that currently I have not enough time to add a test. Do you think it's elaborate? Currently this PR works OK with my code, but I know it's not enough.

**Pull request closed without merging**